### PR TITLE
Allow instructor to skip question

### PR DESCRIPTION
**Pull request by AustinfHunter**
**Summary:**
Added functionality that allows the instructor to skip a question. When a question is skipped, all students associated with the session will have their responses updated with the question_skipped field set to true in order to maintain data integrity and allow for calculations based on responses to handle skipped questions.

**Changes:**
1. Added the skip_question_db method to the InstructorConsumer class. This handles updating data related the question to include a skipped status, locks the question, and creates skipped responses for all students associated with the current quiz session.
2. Added the skip_question method to the InstructorConsumer class. This sends a response to the instructor including the number of student responses updated.

**Examples:**
Instructor skipping question
successful_lock

Student response fails when question has been skipped
student_fails_afer_lock

### CODE REVIEW

**Review by amhajja — APPROVED** [review]
Very nice, Austin; thank you for making the requested change(s)!

File: api/consumers.py [hunk 1]
```diff
@@ -251,6 +253,19 @@ async def add_to_duration(self, question_id, extension: int):
...
+        quiz_session_question = QuizSessionQuestion.objects.get(
+            question__id=question_id, quiz_session__code=self.code
+        )
+        quiz_session_question.skipped = True
+        quiz_session_question.unlocked = False
+        quiz_session_question.save()
+
+    async def skip_question(self, question_id):
+        await self.skip_question_db(question_id)
+        await self.send_next_question()
```
Comment (amhajja) [inline api/consumers.py:267]: @aaladaruncc, could you please include a Postman screenshot showing this?

**Review by amhajja — CHANGES_REQUESTED** [review]
@aaladaruncc, left a minor comment/request.

**Comment by aaladaruncc** [comment]
> @aaladaruncc, left a minor comment/request.

Here is the screenshot, I have also attached to the review:


image

**Review by amhajja — APPROVED** [review]
Looks great, thank you @AustinfHunter an @aaladaruncc!

**Pull request merged**